packages/coinstac-ui/app/render/components/maps/maps-edit.jsx

```diff
@@ -266,11 +268,20 @@ class MapsEdit extends Component {
     const cons = this.state.activeConsortium;
     this.props.saveAssociatedConsortia(cons);
     const runs = this.props.userRuns;
+    const currentUserId = this.props.auth.user.id;
+
+    let mappedForRun = cons.mappedForRun || [];
+
+    if (mappedForRun.indexOf(currentUserId) === -1) {
+      mappedForRun = [...mappedForRun, currentUserId]
+    }
 
     this.props.getCollectionFiles(cons.id)
       .then((filesArray) => {
         this.setState({isMapped: true});
 
+        this.props.saveConsortium({ id: cons.id, name: cons.name, description: cons.description, mappedForRun });
```

Comment:
no need to send name and description here

Reply:
@rochaeb , name and description are required fields for consortia

Reply:
So I'd say create another mutation for this, to improve readability. Using this mutation as is can be misleading as you're sending unnecessary stuff...

Reply:
@rochaeb , do you think creating another mutation is better than sending unnecessary info?
what about you, @rssk ?

Reply:
yeah, I think so

Reply:
@rochaeb , it is already done